### ISSUE TITLE
rnn/Sequencer.lua:90: attempt to index field 'tableoutput' (a nil value)

### ISSUE THREAD

**Issue by tim9510019** [Element-Research/rnn#420]
I tried to evaluate the LSTM model, but the error message "nn/Sequencer.lua:90: attempt to index field 'tableoutput' (a nil value) " always shows up.

[net] loading model ../uni_image_np_50.t7	
nn.Sequencer @ nn.Recursor @ nn.MaskZero @ nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> (14) -> (15) -> (16) -> (17) -> (18) -> (19) -> (20) -> (21) -> output]
  (1): cudnn.SpatialConvolution(4 -> 32, 3x3, 2,2)
  (2): nn.SpatialBatchNormalization (4D) (32)
  (3): cudnn.ReLU
  (4): cudnn.SpatialConvolution(32 -> 64, 3x3, 2,2)
  (5): nn.SpatialBatchNormalization (4D) (64)
  (6): cudnn.ReLU
  (7): nn.SpatialDropout(0.400000)
  (8): cudnn.SpatialConvolution(64 -> 128, 3x3, 2,2)
  (9): nn.SpatialBatchNormalization (4D) (128)
  (10): cudnn.ReLU
  (11): nn.SpatialDropout(0.400000)
  (12): nn.Reshape(1152)
  (13): nn.Linear(1152 -> 512)
  (14): nn.BatchNormalization (2D) (512)
  (15): cudnn.ReLU
  (16): nn.Dropout(0.5, busy)
  (17): nn.Linear(512 -> 512)
  (18): nn.LSTM(512 -> 512)
  (19): nn.Dropout(0.5, busy)
  (20): nn.Linear(512 -> 13)
  (21): cudnn.LogSoftMax
}
/home/timchen/torch/install/bin/luajit: /home/timchen/torch/install/share/lua/5.1/rnn/Sequencer.lua:90: attempt to index field 'tableoutput' (a nil value)
stack traceback:
	/home/timchen/torch/install/share/lua/5.1/rnn/Sequencer.lua:90: in function 'forward'
	./net/rnntrain.lua:28: in function 'batchEval'
	./net/train.lua:24: in function 'epochEval'
	./net/train.lua:46: in function 'train'
	net/main.lua:45: in main chunk
	[C]: in function 'dofile'
	...chen/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50

**Comment by tastyminerals** [comment]
`tableoutput` gets filled when `Sequencer:updateOutput` method is called, during forward pass. If that is not happening then I would check my model `input` and see if I am actually feeding data into it.

**Comment by Acciwu** [comment]
I am reproducing the code of [deep-soli radar], the same problem you encountered, if rnn uses version [torch/rnn] and appears [attempt to call field 'Recurrent' (a nil value)], replace it with [ Element-Research/rnn] has the same problem as you [rnn/Sequencer.lua:90: attempt to index field 'tableoutput' (a nil value) ]

Did you solve this problem please?
